python/vyos/ifconfig/interface.py

```diff
@@ -59,6 +59,36 @@
 
 link_local_prefix = 'fe80::/64'
 
+
+def _interface_exists_in_netns(interface_name, netns):
+    from vyos.util import rc_cmd
+    rc, out = rc_cmd(f'ip netns exec {netns} ip link show dev {interface_name}')
+    if rc == 0:
+        return True
+    return False
+
+def _is_intf_addr_assigned(ifname, addr, netns=None):
```

Comment:
Why not use the `vyos.validate.is_intf_addr_assigned()` function end add an optional netns parameter?

Reply:
The current original module uses `netifaces`, `ifaddresses` and `AF_INET` https://github.com/vyos/vyos-1x/blob/6dd683f3e7a966bf0124444b746ab4a223bbe104/python/vyos/validate.py#L54-L56
I think it doesn't work with `netns`, so I added my own function

Reply:
Then please change the base function so we have only one instance of that function

Reply:
It should be in a separate commit, as required container build changes.
Due to `jmespath` not existing, it  cannot make the deb pkg
```

======================================================================
ERROR: test_is_addr_assigned (test_validate.TestVyOSValidate.test_is_addr_assigned)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vyos/work/T5241-control-edition/vyos-1x/src/tests/test_validate.py", line 25, in test_is_addr_assigned
    self.assertTrue(vyos.validate.is_addr_assigned('127.0.0.1'))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/vyos/work/T5241-control-edition/vyos-1x/python/vyos/validate.py", line 87, in is_addr_assigned
    if is_intf_addr_assigned(interface, ip_address):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/vyos/work/T5241-control-edition/vyos-1x/python/vyos/validate.py", line 52, in is_intf_addr_assigned
    import jmespath
ModuleNotFoundError: No module named 'jmespath'

```

Reply:
Can you send the PR for container update?

Reply:
Sure https://github.com/vyos/vyos-build/pull/354

Reply:
> Then please change the base function so we have only one instance of that function

Done!